.gitignore

```diff
@@ -78,4 +78,5 @@ celerybeat-schedule
 .env
 
 # Spyder project settings
-.spyderproject
\ No newline at end of file
+.spyderprojectvenv
+venv
```

Comment:
Is this meant to be `.spyderproject` replaced by `.spyderprojectvenv` and `venv` added?

Or just `venv` added?